### PR TITLE
Update stories for FlexLayout and StackLayout

### PR DESCRIPTION
**Pull request by Fercas123**
delete unused example for stack layout and turn on flex wrap by default for example's accessibility

### CODE REVIEW

File: packages/core/src/__tests__/__e2e__/layout/Flex.cy.tsx [hunk 1]
```diff
@@ -59,11 +59,11 @@ describe("GIVEN a Flex", () => {
     });
   });
 
-  describe("WHEN wrap is set to true", () => {
-    it("THEN it should render with flex wrap", () => {
+  describe("WHEN wrap is set to false", () => {
+    it("THEN it should render with no flex wrap", () => {
       cy.mount(<DefaultFlexLayout wrap />);
```
Comment (joshwooding) [inline packages/core/src/__tests__/__e2e__/layout/Flex.cy.tsx:64]: This line needs to be updated.

File: packages/core/stories/layout/split-layout.stories.tsx [hunk 2]
```diff
@@ -46,18 +34,15 @@ export default {
...
 );
 
 const rightItem = (
   <FlowLayout>
-    <ContentBlock>Item 4</ContentBlock>
-    <ContentBlock>
-      Item
-      <br />5
-    </ContentBlock>
+    <FlexContent classname="layout-content-right">Item 4</FlexContent>
```
Comment (joshwooding) [inline packages/core/stories/layout/split-layout.stories.tsx:44]: We should fix the className capitalization on FlexContent instead

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Review by joshwooding — DISMISSED** [review]

File: packages/core/src/layout/SplitLayout/SplitLayout.tsx [hunk 3]
```diff
@@ -68,6 +68,7 @@ export const SplitLayout = forwardRef<HTMLDivElement, SplitLayoutProps>(
   ) {
     return (
       <FlexLayout
+        align={align}
```
Comment (joshwooding) [inline packages/core/src/layout/SplitLayout/SplitLayout.tsx:71]: Is it worth calling out this bug fix?

File: .changeset/cold-balloons-hug.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@jpmorganchase/uitk-core": minor
```
Comment (joshwooding) [inline .changeset/cold-balloons-hug.md:2]:
```suggestion
"@jpmorganchase/uitk-core": patch
```

**Review by ZarkoPernar — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**